site/content/docs/commands/deploy.en.md

```diff
@@ -7,31 +7,44 @@ $ copilot deploy
 
 This command is used to run either [`copilot svc deploy`](../commands/svc-deploy.en.md) or [`copilot job deploy`](../commands/job-deploy.en.md) under the hood. The steps involved in `copilot deploy` are the same as those involved in `copilot svc deploy` and `copilot job deploy`:
 
-1. When `image.build` exists in the manifest:
+1. If your service does not exist, optionally initialize it (via `--init-env[=false]`).
+2. If the target environment does not exist, optionally initialize it with custom credentials (via `--init-env[=false]`, `--region`, `--profile`, or `--aws-*` flags).
+3. Optionally deploy the environment before service deployment (via `--deploy-env[=false]`).
+4. When `image.build` exists in the manifest:
     1. Build your local Dockerfile into an image
     2. Tag it with the value from `--tag` or the latest git sha (if you're in a git directory)
     3. Push the image to ECR
-2. Package your manifest file and addons into CloudFormation
-3. Create / update your ECS task definition and job or service.
+5. Package your manifest file and addons into CloudFormation.
+6. Create / update your ECS task definition and job or service.
 
 ## What are the flags?
 
 ```
       --allow-downgrade                Optional. Allow using an older version of Copilot to update Copilot components
                                        updated by a newer version of Copilot.
   -a, --app string                     Name of the application.
+      --aws-access-key-id string       Optional. An AWS access key for the environment account.
+      --aws-secret-access-key string   Optional. An AWS secret access key for the environment account.
+      --aws-session-token string       Optional. An AWS session token for temporary credentials.
+      --deploy-env                     Deploy the target environment before deploying the workload.
       --detach                         Optional. Skip displaying CloudFormation deployment progress.
   -e, --env string                     Name of the environment.
       --force                          Optional. Force a new service deployment using the existing image.
+                                       Not available with the "Static Site" service type.
   -h, --help                           help for deploy
+      --init-env                       Confirm initializing the target environment if it does not exist.
+      --init-wkld                      Optional. Initialize a workload before deploying it.
   -n, --name string                    Name of the service or job.
-      --no-rollback bool               Optional. Disable automatic stack
+      --no-rollback                    Optional. Disable automatic stack 
```

Comment:
just curious: was this wrong before or did we change it?

Reply:
nvm I now see it was just wrong before! thx for fixing.

Reply:
No i guess it's not published yet.

Reply:
actually you are right. it was wrong before lol